subsys/power/device.c

```diff
@@ -29,6 +29,12 @@ static const char *const core_devices[] = {
 	"sys_clock",
 	"UART_0",
 };
+#elif defined(CONFIG_SOC_SERIES_CC13X2_CC26X2)
+#define MAX_PM_DEVICES	15
+static const char *const core_devices[] = {
+	"sys_clock",
+	"UART_0",
+};
```

Comment:
we need a better way to deal with SOC related data, should not be in core implementation... But this is a story for another time.